modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCenter.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.credential;
+
+import java.util.List;
+import org.eclipse.xpanse.modules.models.enums.Csp;
+
+/**
+ * The credential center.
+ */
+public class CredentialCenter {
+
+    /**
+     * Get the credential abilities of Csp.
+     *
+     * @param csp The csp to get.
+     */
+    List<Credential> getCredentialAbility(Csp csp) {
+        return null;
+    }
+
+    /**
+     * Create credential for the @Csp with @userName.
+     *
+     * @param csp        The csp  to create for.
+     * @param userName   The userName to create for.
+     * @param credential The credential to create.
+     */
+    boolean createCredential(Csp csp, String userName, Credential credential) {

```

Comment:
Should the plugin call this method multiple times to store different Credential variables?

Reply:
I don't think so, we only need one kind of credential available for per-csp (maybe per-csp-user).